docs/tutorial/lambda-aws.txt

```diff
@@ -0,0 +1,121 @@
+============================================
+Deploy to Serverless AWS Lambda with BrefPHP
+============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. versionadded:: 1.17
+
+Overview
+--------
+
+`Bref <https://bref.sh>` allows to deploy serverless PHP applications on AWS Lambda.
+In this tutorial, you will deploy a simple PHP application with the MongoDB PHP extension,
+and connect to an Atlas cluster using AWS IAM authentication.
+
+Pre-requisites
+--------------
+
+Before you begin, you must install bref on your machine. You can follow the
+`official documentation to setup bref <https://bref.sh/docs/setup>`_.
+
+Install the mongodb extension
+-----------------------------
+
+By default, the bref layer is compiled with PHP and a few extensions. Additional extensions
+are provided in additional layers.
+
+Start by creating a new directory for your project and install the required mongodb
+and bref dependencies. This project will be a bare minimum PHP web application that
+connects to a MongoDB cluster, no framework is used, but you can adapt the code to
+your needs.
+
+.. code-block:: bash
+    mkdir bref-mongodb-app && cd bref-mongodb-app
+    composer init
+    composer require bref/bref bref/extra-php-extensions
+    vendor/bin/bref init
+
+
+A very simple web page have been created. To validate the deployment, you can start
+by deploying this default application.
+
+.. code-block:: bash
+    serverless deploy
+
+BrefPHP provides a Lambda layer with PHP and some very common extensions.
+Additional extensions are provided by the package `bref/extra-php-extension <https://github.com/brefphp/extra-php-extensions>`.
+
+.. code-block:: yaml
+
+    plugins:
+        - ./vendor/bref/bref
+        - ./vendor/bref/extra-php-extensions
+
+    functions:
+        api:
+            handler: index.php
+            runtime: php-83-fpm
+            layers:
+                - ${bref-extra:mongodb-php-83}
+
+
+Let's try to use the MongoDB driver with this simple web page that list planets
+from the `sample dataset <https://www.mongodb.com/docs/atlas/sample-data/>`.
```

Comment:
I need to fix the doc link.

Reply:
Since this is RST syntax, shouldn't you have a double underscore after the link for an anonymous reference? For example:

```
`sample dataset <https://www.mongodb.com/docs/atlas/sample-data/>`__
```

Reply:
I think there is a prefix for links to the docs.